projects/demo/src/environments/environment.prod.ts

```diff
@@ -1,6 +1,4 @@
 export const environment = {
     production: true,
     ym: 87890624,
-    playground:
-        'https://stackblitz.com/~/github.com/taiga-family/taiga-playground?file=src/app/app.component.html',
```

Comment:
A good full-fledged repository is also great to have, I wish we could have this link somewhere in the docs too.